### PR TITLE
Update the release config

### PR DESCRIPTION
**Pull request by doomspork**

### CODE REVIEW

File: config/releases.exs [hunk 1]
```diff
@@ -1,16 +1,23 @@
 import Config
 
-producer_config =
-  "SQS_PRODUCER"
+config =
```
Comment (doomspork) [inline config/releases.exs:3]: I think we should just get in the habit of using `config` for this variable so when we ultimately do copy+paste between projects we don't forget to rename these variable. That has caught us a couple times already

File: config/releases.exs [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+  "CONFIG"
   |> System.fetch_env!()
   |> Jason.decode!()
 
 config :copy_cat,
   producer:
     {BroadwaySQS.Producer,
-     queue_url: producer_config["queue_url"],
+     queue_url: config["COPYCAT_SQS_URL"],
```
Comment (doomspork) [inline config/releases.exs]: @btkostner should we keep this ENV name specific to the app or standardize on `SQL_URL` instead?

Comment (btkostner) [reply config/releases.exs]: ummm, I think we are going to just have a single queue url for each project. So probably `SQL_URL`. Keeps it easier when we create new projects

**Review by btkostner — APPROVED** [review]

File: config/releases.exs (hunk 2 above)
Comment (doomspork) [inline config/releases.exs]:
```suggestion
     queue_url: config["SQS_URL"],
```

**Pull request merged**